src/ser.rs

```diff
@@ -267,7 +267,13 @@ impl<'s> ser::Serializer for &'s Serializer {
         serialize_str(&str);
     }
 
-    // TODO: we might want to support `BigInt` here in the future.
+    #[cfg(feature="bigint_serialization")]
```

Comment:
Rather than making this a compile-time feature, that will affect all usages of `serde-wasm-bindgen` across the entire crate tree, can you make this a runtime option on `Serializer` like there is already `serialize_maps_as_objects`? It would allow each user to decide which format they prefer.